tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -649,21 +649,9 @@ def test_with_go(self, runtime, code_uri, mode):
                 self.built_template, self.FUNCTION_LOGICAL_ID, self._make_parameter_override_arg(overrides), expected
             )
 
-        self.verify_docker_container_cleanedup(runtime)
-
-    @parameterized.expand([("go1.x", "Go")])
-    @skipIf(SKIP_DOCKER_TESTS, SKIP_DOCKER_MESSAGE)
-    @pytest.mark.flaky(reruns=3)
-    def test_go_must_fail_with_container(self, runtime, code_uri):
-        use_container = True
-        overrides = {"Runtime": runtime, "CodeUri": code_uri, "Handler": "hello-world"}
-        cmdlist = self.get_command_list(use_container=use_container, parameter_overrides=overrides)
-
-        LOG.info("Running Command: {}".format(cmdlist))
-        process_execute = run_command(cmdlist, cwd=self.working_dir)
-
-        # Must error out, because container builds are not supported
-        self.assertEqual(process_execute.process.returncode, 1)
+        if use_container:
+            self.verify_docker_container_cleanedup(runtime)
```

Comment:
Something to check, are we using public ecr or dockerhub?

Reply:
Checked on L654 by:

https://github.com/aws/aws-sam-cli/blob/a2821662642d229f34c212d097c9dbf8f0351883/tests/integration/buildcmd/build_integ_base.py#L147